packages/contentstack-export/src/export/modules-js/assets.js

```diff
@@ -282,9 +282,11 @@ module.exports = class ExportAssets {
       }
 
       if (version <= 0) {
-        const assetVersionInfoFile = path.resolve(self.assetsFolderPath, uid, '_contentstack_' + uid + '.json');
-        fileHelper.writeFileSync(assetVersionInfoFile, assetVersionInfo);
-        return resolve();
+        if(validateUids(uid)){
+          const assetVersionInfoFile = path.resolve(sanitizePath(self.assetsFolderPath), sanitizePath(uid), '_contentstack_' + sanitizePath(uid) + '.json');
+          fileHelper.writeFileSync(assetVersionInfoFile, assetVersionInfo);
```

Comment:
@cs-raj , do we need to sanitize uid?

Reply:
Yes, everything which we are passing into the resolve or join we need to sanitize